src/components/ChatPopUp.tsx

```diff
@@ -88,6 +88,8 @@ export function ChatPopUp(props: ChatPopUpProps) {
     showRestartButton = true,
     onClose: customOnClose,
     title,
+    footer,
+    header,
```

Comment:
thinking about it more, this wouldn't work because the header component needs to take in the `onClose` function below in order to control the display behavior of the chat popup/panel. We would need to take in a header component type and construct it inside.